dask_geopandas/core.py

```diff
@@ -336,6 +337,38 @@ def hilbert_distance(self, p=15):
 
         return distances
 
+    def morton_distance(self, p=15):
+
+        """
+        Calculate distance of geoms along Morton curve
+
+        The Morton curve is also known as Z-order https://en.wikipedia.org/wiki/Z-order
+
+        Parameters
+        ----------
+
+        p : int
+            precision of the Morton curve
+
+        Returns
+        ----------
+        type : pandas.Series
+            Series containing distances from Morton curve
```

Comment:
It returns dask Series, not pandas Series.

Reply:
updated in a08e85e